bin/get_metadata.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import xmltodict
+from collections import OrderedDict
+import re
+import argparse
+import os
+import json
+
+
+class RunfolderInfo():
+
+    def __init__(self, runfolder):
+        self.runfolder = runfolder
+        self.run_parameters = self.read_run_parameters()
+        self.stats_json = self.read_stats_json()
+        self.description_and_identifier = OrderedDict()
+        self.run_parameters_tags = \
+            {'RunId': 'Run ID', 'RunID': 'Run ID',
+             'ApplicationName': 'Control software', 'Application': 'Control software',
+             'ApplicationVersion': 'Control software version',
+             'Flowcell': 'Flowcell type', 'FlowCellMode': 'Flowcell type',
+             'ReagentKitVersion': 'Reagent kit version',
+             'RTAVersion': 'RTA Version', 'RtaVersion': 'RTA Version',
+             }
+
+    def find(self, d, tag):
+        if tag in d:
+            yield d[tag]
+        for k, v in d.items():
+            if isinstance(v, dict):
+                for i in self.find(v, tag):
```

Comment:
This can be simplified a little bit by using `yield from` syntax. It would look something like this:
```
if isinstance(v, dict):
    yield from self.find(v, tag)
```

Reply:
Done!